examples/stats/web_client_monitoring/app.js

```diff
@@ -0,0 +1,131 @@
+ * with the opencensus API.
+ */
+
+"use strict";
+
+const express = require("express");
+const assert = require('assert');
+const process = require("process");
+const bodyParser = require('body-parser');
+const { Stats, MeasureUnit, AggregationType } = require('@opencensus/core');
+const { StackdriverStatsExporter } = require('@opencensus/exporter-stackdriver');
+
+const app = express();
+app.use(express.static("web_client"));
+app.use(bodyParser.json());
+
+// The project id must be provided. First, try the local environment
+const project = process.env.GOOGLE_CLOUD_PROJECT;
+assert(typeof project !== 'undefined' && project,
+  "Please set environment variable GOOGLE_CLOUD_PROJECT to the project id.");
+console.log(`Sending metrics data to project: ${project}`);
+
+// OpenCensus setup
+const stats = new Stats();
+const exporter = new StackdriverStatsExporter({projectId: project});
+stats.registerExporter(exporter);
+const mDNSTimeMs = stats.createMeasureDouble("web_client/dnsTime",
+                       MeasureUnit.MS,
+                       "DNS lookup time");
+const mConnectTimeMs = stats.createMeasureDouble("web_client/connctTime",
+                       MeasureUnit.MS,
+                       "TLS negotiation time");
+const mLatencyMs = stats.createMeasureDouble("web_client/totalTime",
+                       MeasureUnit.MS,
+                       "Total page loading time");
+const mClickCount = stats.createMeasureInt64("web_client/clickCount",
+                       MeasureUnit.UNIT,
+                       "Number of clicks");
+const tagKey = "client";
+const dnsTimeView = stats.createView(
+  "webmetrics/dnstime",
+  mDNSTimeMs,
+  AggregationType.DISTRIBUTION,
+  [tagKey],
+  "The distribution of the DNS lookup time",
+  // Bucket Boundaries:
+  [0, 25, 50, 75, 100, 200, 400, 600, 800, 1000, 2000, 4000, 6000]
+);
+const connectTimeView = stats.createView(
+  "webmetrics/connecttime",
+  mConnectTimeMs,
+  AggregationType.DISTRIBUTION,
+  [tagKey],
+  "TLS negotiation time",
+  // Bucket Boundaries:
+  [0, 25, 50, 75, 100, 200, 400, 600, 800, 1000, 2000, 4000, 6000]
+);
+const latencyView = stats.createView(
+  "webmetrics/latency",
+  mLatencyMs,
+  AggregationType.DISTRIBUTION,
+  [tagKey],
+  "The distribution of the latencies",
+  [0, 25, 50, 75, 100, 200, 400, 600, 800, 1000, 2000, 4000, 6000]
+);
+const clickCountView = stats.createView(
+  "webmetrics/click_count",
+  mClickCount,
+  AggregationType.COUNT,
+  [tagKey],
+  "The number of lines from standard input"
+);
+
+// Process the metrics data posted to the server
+app.post("/metrics", (req, res) => {
+  let dnsTime = req.body["dnsTime"];
+  let connectTime = req.body["connectTime"];
+  let totalTime = req.body["totalTime"];
+  let clickCount = req.body["count"];
+  console.log(`totalTime ${totalTime}`);
+  const tags = { "client": "web" };
+  console.log(`dnsTime ${dnsTime}`);
+  stats.record({
```

Comment:
I made a comment about using, mLatencyMs and a bunch of tags, PTAL.

Reply:
I have changed code according to in the previous commit.